### ISSUE TITLE
TalkBack  focus at incorrect location (V46)

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-alert-app#656]
When the user goes from Step 5 to Step 6 at on-boarding screens with TalkBack on, "Turn on COVID-19 Exposure Notification" pops up. When the user selects Cancel OR Turn on, the focus goes to Done at the bottom right the first time they come to this. When they go Back and come back to this page again, then it goes to top like it should. But sometimes it goes to Done. Tried is more than 5 times after reinstalling the app.

Android 10 on OnePlus 7
Android 8.1.0 is fine

![image](https://user-images.githubusercontent.com/14280062/87319271-66453800-c4f7-11ea-81c9-9c97d843f3c2.png)

![image](https://user-images.githubusercontent.com/14280062/87319635-e10e5300-c4f7-11ea-9a36-b69a17da5aa8.png)

**Comment by sarasuri** [comment]
Samsung Galaxy J3 Prime
Android version 7.0
App version: 1.0 (46)

Similar issue on Android 7, the focus goes to "Done" button not to the top of the screen.

**Comment by jelen-HC** [comment]
I am unable to re-create this issue; I have tested this via Android 7 & 8 as well as on the iPhone X. It has worked as intended every time I have tested it with the latest build.

**Comment by henrytao-me** [comment]
Related to this https://github.com/cds-snc/covid-shield-mobile/issues/722

**Comment by sarasuri** [comment]
@jelen-HC 

Samsung Galaxy J3 Prime
Android version 7.0

Tested in v50 with Talkback on and the behaviour mentioned above seems to be inconsistent. Some times the focus stays on "Done" and some times it moves to "Step 5 of 5". Noticed there is slight lag in focus moving to top.

**Comment by sarasuri** [comment]
Samsung Galaxy J3 Prime
Android version 7.0

Samsung Galaxy S5
Android version 6.0

App version: 1.0 (53)

With Talkback on, after going through onboarding flow initially pressing the "Next" button, when users go back and forth between screens pressing the "Back" and "Next" buttons, the focus remains on the buttons instead of moving to the top of screens. 

Expected result
The focus should move to the top of screens.

Screenshots
![focus3](https://user-images.githubusercontent.com/12256067/87712696-12e11d00-c777-11ea-850f-fd42dcfb27b9.gif)
![focus4](https://user-images.githubusercontent.com/12256067/87712701-14124a00-c777-11ea-8d93-5201bd6f9c75.gif)

**Comment by timarney** [comment]
@JuliannaR can you confirm if this is still and issue?
